spec/system/homepage_system_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'Home Page', type: :system do
+  before(:each) do
```

Comment:
obczaj factory_girl i lety

Reply:
zrobione. btw czy factory_girl to nie jest od jakiegoś czasu factory_bot?